README.md

```diff
@@ -398,8 +398,6 @@ Merge multiple objects with merger
 
 #### Enum.from(...args)
 
-#### Enum.NaE()
```

Comment:
I think this should be fixed to show that it is not a method but a property, instead of deleting it.